plugin.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "com.mattermost.custom-attributes",
     "name": "Custom User Attributes",
-    "description": "This plugin allows you to add custom user attributes that show up in the Mattermost UI for users.",
+    "description": "Add custom user attributes that appear in the Mattermost user interface.",
```

Comment:
Feedback on this description welcome. I'm not sure I particularly like `that appear in the Mattermost user interface.`

Reply:
Another option may be

```
Add custom user attributes to Mattermost user profiles.
```

But I'm not sure if the custom user attributes can also appear elsewhere in the Mattermost UI

Reply:
Add custom user attributes to [your/the] Mattermost user profile popover.